pulsar-io/kafka-connect-adaptor/src/main/java/org/apache/pulsar/io/kafka/connect/PulsarOffsetBackingStore.java

```diff
@@ -49,7 +50,7 @@
 @Slf4j
 public class PulsarOffsetBackingStore implements OffsetBackingStore {
 
-    private Map<ByteBuffer, ByteBuffer> data;
+    private ConcurrentHashMap<ByteBuffer, ByteBuffer> data;
```

Comment:
I think it's better to make it `final`. And there is no need to change the type to `ConcurrentHashMap` because we only use the common methods in `Map`.